### DISCUSSION TITLE
Is there a simple way to check for new messages?

### DISCUSSION THREAD

**Discussion by rjolina** [d99kris/nchat#215] [Q&A]
Is there a simpler way to check for new unread messages? The only method I've come up with so far is to periodically run the command `nchat -x ...` and a script that compares the newly generated archive with the old one, showing the changes - that is, new messages. But is there an easier way to do this?

**Comment by d99kris** [comment]
Hi @rjolina - no there's no such support today. There's a feature request #183 reported, so I think we can use that one to track the possible addition of such functionality. I will proceed to close this item.
